### PR TITLE
prevent padding to drop to 0 on large screen on audio detail view

### LINKED ISSUE #1478: Audio detail page does not have horizontal padding on lg screen
## Description

Audio page doesn't have horizontal padding, the content is right on the edges on the `lg` width (1024px).
## Reproduction

1. Set your screen width to 1024px.
2. Open an audio detail page.
3. See error.

## Screenshots

Image of Openverse audio detail page without padding

## Additional context

It would be nice to unify the paddings with the image detail page.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by carlosreyesmunoz**
## Fixes

Fixes #1478 by @obulat 

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by carlosreyesmunoz** [comment]
As a solution, I have removed the rule to set padding to 0 on `lg` screen.
In the description it mentioned to add behaviour like on the image site, but on image site padding is always 4rem, if I apply padding to 4rem on the  `lg` then it creates a very nasty jump from 4rem at 1024px to 1rem at 1023px and smaller. 
if we want same padding than in the images site we should set px-16 (4rem) as on the image site.

File: src/components/VAudioTrack/layouts/VFullLayout.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
       </div>
     </div>
     <div
-      class="flex flex-row flex-wrap lg:flex-nowrap items-top px-4 lg:px-0 lg:max-w-5xl mx-auto gap-6 mt-6"
+      class="flex flex-row flex-wrap lg:flex-nowrap items-top px-4 lg:max-w-5xl mx-auto gap-6 mt-6"
```
Comment (obulat) [inline src/components/VAudioTrack/layouts/VFullLayout.vue:19]:
```suggestion
      class="flex flex-row flex-wrap lg:flex-nowrap items-top px-6 lg:max-w-5xl mx-auto gap-6 mt-6"
```

File: src/pages/audio/_id.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
     </div>
     <VAudioTrack layout="full" :audio="audio" class="main-track" />
     <div
-      class="mt-10 lg:mt-16 flex flex-col gap-10 lg:gap-16 px-4 lg:px-0 lg:max-w-5xl mx-auto"
+      class="mt-10 lg:mt-16 flex flex-col gap-10 lg:gap-16 px-4 lg:max-w-5xl mx-auto"
```
Comment (obulat) [inline src/pages/audio/_id.vue:8]:
```suggestion
      class="mt-10 lg:mt-16 flex flex-col gap-10 lg:gap-16 px-6 lg:max-w-5xl mx-auto"
```

**Review by obulat — APPROVED** [review]
Thank you for your contribution, @carlosreyesmunoz, it works really great!

I've checked the [mockup in Figma](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Audio-Release?node-id=1685%3A20591) and saw that the padding should be at least 24px, ie `px-6`, so I added the suggestions as a code review. You approved and merged them before I finished testing the changes locally 😆

Welcome to Openverse 🎉

**Review by dhruvkb — APPROVED** [review]

**Comment by carlosreyesmunoz** [comment]
Hi, is there anything I can do for the failing checks?

**Pull request merged**